woo-durianpay/woo-durianpay.php

```diff
@@ -207,6 +235,118 @@ public function init_form_fields()
             }
         }
 
+        public function autoEnableWebhook()
+        {
+            $webhookExist = false;
+            $webhookUrl   = esc_url(admin_url('admin-post.php')) . '?action=dp_wc_webhook';
+
+            $key_secret  = $this->getSetting('key_secret');
+            $enabled     = $this->getSetting('enable_webhook');
+
+            //validating the key id and key secret set properly or not.
+            if($key_secret == null)
+            {
+                ?>
+                    <div class="notice error is-dismissible" >
+                     <p><b><?php _e( 'Key Id and Key Secret can`t be empty'); ?><b></p>
+                    </div>
+                <?php
+
+                error_log('Key Id and Key Secret are required to enable the webhook.');
+                return;
+            }
+
+            $eventsSubscribe = $this->getSetting('webhook_events');
+
+            $prepareEventsData = [];
+
+            if(empty($eventsSubscribe) == false)
+            {
+                foreach ($eventsSubscribe as $value) 
+                {
+                    $prepareEventsData[$value] = true;
+                }
+            }
+
+            if(in_array($_SERVER['SERVER_ADDR'], ["127.0.0.1","::1"]))
```

Comment:
127.0.0.1 is this intented ? or used for local testing ?

Reply:
intended